www/templates/user-vhost-needs-authorization

```diff
@@ -0,0 +1,13 @@
+Hi,
+
+We'd be happy to get that set up for you, but we require
+these requests to be authorized by a existing signatory.
+For {group}, we see the following people as signatories:
+
+{signatories}
```

Comment:
This might be difficult to fill out as is since you can't type a newline in a text box

Reply:
We should either add functionality for a multiline text box, or just put in a big "REPLACE THIS" in all caps so staffers know to replace it with the signat output.

Reply:
Multiline text box would be my vote here

Reply:
With a multiline text box, we could also retrofit the Apache security template to make use of that (if we don't get rid of it soon)